chapters/offenses.adoc

```diff
@@ -40,6 +48,8 @@ NOTE: Dribblers can still be used to dribble large distances with the ball as lo
 ==== Ball Speed
 A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
 
+Violations do not stop game play. Goals scored with kicks exceeding the ball speed limit will not be counted. Increase <<Fouls, foul counter>> by one.
```

Comment:
How is gameplay restarted after a goal is not counted?

Reply:
Good point. It is defined in the "Scoring goals" sections for other invalid goals: It is counted as if the ball crossed the goal line outside of the goal. A reference to this sections should probably be sufficient, but I haven't checked the formulations in detail.